### DISCUSSION TITLE
Hello,

### DISCUSSION THREAD

**Discussion by patrik-upspot** [fengyuanchen/viewerjs#499] [General]
I use the viewer.js with angular 10. all is Fine, but I have one big problem. I have some thumbnails, but the original photos should be load on click. They loaded by the angular backend, because they need a authorization header. I'm not able to using the viewer.js in this scenario. Do you have any idea how I can reload the pictures? A viewer.update() after reload does not work. 

Thanks for your help.

**Comment by patrik-upspot** [comment]
I've testet a little bit. I load the picture in original size by clicking the Thumbnail. At this moment the viewer opens the modal view. Now i cannot update the viewer. If i change the picture before open the viewer, all works fine, but thats not my workflow. Is it possible to change the src of the image after the modal dialog opens?

**Reply by fengyuanchen** [reply]
No support for that.
